### PR TITLE
Send Mail using Queue

### PR DESCRIPTION
**Pull request by sondth-1037**
## Related Tickets:
- [#27880](https://edu-redmine.sun-asterisk.vn/issues/27880)

## WHAT this PR does?
- Send Mail to Admin using Queue

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by sondth-1037** [comment]
ready

File: app/Console/Commands/AdminFilterRequest.php [hunk 1]
```diff
@@ -97,19 +98,19 @@ public function handle()
...
             }
 
-            if ($email['file']) {
-                Storage::delete(config('constance.excel.path') . $this->email['file']->getFile()->getFilename());
+            if (array_key_exists('file', $email)) {
+                Storage::delete(config('constance.excel.path') . $email['file']->getFile()->getFilename());
             }
         } catch (Exception $e) {
-
+            var_dump("Error". $e);
```
Comment (huyenttm-0901) [inline app/Console/Commands/AdminFilterRequest.php]: lỗi em ghi vào log nhé, ko in ra đâu

Comment (sondth-1037) [reply app/Console/Commands/AdminFilterRequest.php]: vâng ạ

**Comment by sondth-1037** [comment]
ready

**Pull request merged**